### ISSUE TITLE
Cannot hide trainning output in jupyter notebook.

### ISSUE THREAD

**Issue by Enpen** [DataCanvasIO/HyperGBM#51]
**System information**
- OS Platform and Distribution (e.g., CentOS 7.6): win10
- Python version:3.6.7
- HyperGBM version:0.2.3
- Other Python packages(run `pip list`):

**Describe the current behavior**
if i set log_level='error' in make_experiment, no trainning output when i run it on local PC, However, if i run it in Jupyter notebook, it useless, i try all log_lovels, output is same.

**Describe the expected behavior**
if  set log_level='error',  error info  only can be showed when i run in jupyter notebook.

**Standalone code to reproduce the issue**
import pandas as pd
from sklearn import datasets
from sklearn.model_selection import train_test_split
X,y = datasets.load_breast_cancer(as_frame=True,return_X_y=True)
X_train,X_test,y_train,y_test = train_test_split(X,y,train_size=0.7,random_state=335)

from hypergbm import make_experiment
train_data = pd.concat([X_train,y_train],axis=1)
experiment = make_experiment(train_data, target='target', log_level = 'error',reward_metric='precision')
estimator = experiment.run()

**Comment by lixfz** [comment]
HyperGBM setup progressive output in notebook with experiment callback and HyperModel search callback by default. 
To disable all output, set `callbacks`option  and `search_callbacks` option  to empty list, eg:

```python
exp=make_experiment(..., callbacks=[], search_callbacks=[])
```
